bgpd/bgp_rpki.c

```diff
@@ -1036,13 +1050,30 @@ static void free_cache(struct cache *cache)
 	XFREE(MTYPE_BGP_RPKI_CACHE, cache);
 }
 
+static int bgp_rpki_write_debug(struct vty *vty, bool running)
+{
+	if (rpki_debug_conf && running) {
+		vty_out(vty, "debug rpki\n");
+		return 1;
+	}
+	if ((rpki_debug_conf || rpki_debug_term) && !running) {
+		vty_out(vty, "  BGP RPKI debugging is on\n");
```

Comment:
We should ensure this is omitted if we turn off `no debug rpki`.

Reply:
Now OK
```
r2# debug rpki 
r2# show  debugging 
MGMT debugging status:

Zebra debugging status:

BGP debugging status:
  BGP RPKI debugging is on

Staticd debugging status

r2# no debug rpki 
r2# show  debugging 
MGMT debugging status:

Zebra debugging status:

BGP debugging status:

Staticd debugging status

```